api/services/administration/general/schemas/setCloudCredentialsForAWS.yaml

```diff
@@ -9,8 +9,3 @@ properties:
         description: Provide the IAM Role ARN connected to another AWS account 922761411349 and with the latest Spot Policy - https://docs.spot.io/administration/api/spot-policy-in-aws
         example: arn:aws:iam::1234567890:role/Spot_Iam_Role
         required: true
-      externalId:
-        type: string
-        description: Provide the External ID used while setting up the IAM Role. This should be a random string of your choice.
```

Comment:
maybe just re-write the description?

Reply:
Now after the ui deployed its changes we can remove it